packages/ui-date-time-input/src/DateTimeInput/README.md

```diff
@@ -45,7 +45,7 @@ class Example extends React.Component {
 render(<Example />)
 ```
 
-#### A required DateInput with `stacked` layout that warns if the value in the past:
+#### A required DateInput with `stacked` layout that warns if the value in the past that allows the user to enter any time value:
```

Comment:
This sentence doesn't really make sense :) Can you please rewrite it a bit? Also, it might be too long for a title, you could add more detailed info in a paragraph below.